packages/codemods/meteor/3/mongo-db-async-methods/cdmd_dist/index.cjs

```diff
@@ -0,0 +1,47 @@
+/*! @license
```

Comment:
should this be checked into git?

Reply:
for some reason it got exported by studio. will remove.
cc @DmytroHryshyn